org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/launchConfigurations/LaunchConfigurationView.java

```diff
@@ -442,6 +443,7 @@ public void keyPressed(KeyEvent e) {
 		}
 		fFilteredNotice = SWTFactory.createLabel(parent, IInternalDebugCoreConstants.EMPTY_STRING, 1);
 		fFilteredNotice.setBackground(parent.getBackground());
+		((GridData) fFilteredNotice.getLayoutData()).grabExcessHorizontalSpace = true;
```

Comment:
SWTFactory.createLabel() explictily sets gd.grabExcessHorizontalSpace = false. So, instead of accessing GridaData outside and changing it to true here (Line 446), can we have a new SWTFactory.createLabel() call that takes grabExcessHorizontalSpace as a parameter and is called with true. @SarikaSinha What do you suggest?

Reply:
Thanks @lshanmug 
@elsazac Can you try SWTFactory.createLabel() ?

Reply:
@lshanmug @SarikaSinha 

Addressed the requested changes.Could you review this.?Please Let me know if anything else is required on this.